modules/ROOT/pages/introduction/intro-config-use-fd.adoc

```diff
@@ -0,0 +1,142 @@
+= Use Flow Designer to Configure a Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Flow Designer is a tool in Anypoint Design Center that enables you to create apps visually. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To use Flow Designer to configure a connector:
+
+. Configure the input source (trigger) for the flow.
+. Configure the target connector operation.
+
+[[configure-input-source]]
+== Configure the Input Source for the Flow
+
+. Log into Anypoint Platform.
+. Click *Start designing*.
+. In Design Center, click *Create new*.
+. Click *Create new application*.
+. Enter a project name and click *Create*.
+. Click *Go straight to canvas* to exit from *Let's get started*.
```

Comment:
It's odd to me that the two steps that I commented on above are for creating an app that consists of two connectors; yet, in this wizard which is for doing just that, you tell users to click "Go straight to canvas". If you are telling users how to create an app that consists of two connectors, why not have them use the wizard?